pagopa/io-app
# chore(IT Wallet): [SIW-432] Update `ItwDiscoveryInfoScreen` with new DS

### 🚧 This PR dependes on #5173 

## Short description
This PR updates the Discovery flow (and related action banner) to the new DS components.

https://github.com/pagopa/io-app/assets/49342144/6a9a2bc8-ef30-4def-abaa-bad163be5333

## List of changes proposed in this pull request
- Update `ItwTextInfo` to add support for H6 and H2 header level
- Add new images
- Update `ItwDiscoveryInfoScreen.tsx`

To better organize the locale string it is necessary to also update `ItwCredentialIssuingInfoScreen` and `ItwPresentationDataScreen`. The tos string has been moved as an itWallet child.

## How to test

1. Start the app
2. Enable expiremental DS from Profile section
3. In Home message and Wallet Home should be the new Action banner.
4. Tap on the action banner
5. the new Discovery info screen should now use last DS components
